content/en/docs/concepts/services-networking/_index.md

```diff
@@ -30,31 +30,32 @@ The Kubernetes network model is built out of several pieces:
       * Agents on a node (e.g. system daemons, kubelet) can communicate
         with all pods on that node.
 
-  * The [`Service`](/docs/concepts/services-networking/service/) API
-    lets you provide a stable "cluster IP" for a service implemented
+  * The [Service](/docs/concepts/services-networking/service/) API
+    lets you provide a stable (long lived) IP address or hostname for a service implemented
```

Comment:
what is the "long lived" meaning here? it can be short lived, is about is `reserved` ? or something like that?

Reply:
It can outlive one Pod, is what I want to convey.